pkg/router/functionHandler.go

```diff
@@ -309,8 +313,21 @@ func (roundTripper *RetryingRoundTripper) RoundTrip(req *http.Request) (*http.Re
 			dumpReqFunc(newReq)
 		}
 
+		openTracingEnabled, err := strconv.ParseBool(os.Getenv("OPENTRACING_ENABLED"))
```

Comment:
We should avoid reading the env variable in the request path. Can we read it once and pass to all round-trippers created?

Reply:
Also if we get an error while reading here, it's not fatal req per say so we can avoid returning.